web/src/resize.js

```diff
@@ -145,8 +145,19 @@ export function resize_sidebars() {
     return h;
 }
 
+export function reposition_message_header() {
+    // Since `navbar_alerts_wrapper`'s height can vary based on text / language, we
+    // need to adjust at what `top` position do `message-header` becomes `sticky`.
+    // Best way to do this is via adding custom CSS to the DOM instead of running endless
+    // javascript queries to find and update them on various re-renders.
+    const navbar_fixed_height = $("#navbar-fixed-container").safeOuterHeight(true);
+    const style = document.querySelector("#sticky_message_header_styles");
+    style.textContent = `.message_list .message_header { top: ${navbar_fixed_height}px !important; }`;
+}
```

Comment:
We could probably clean this up to involve a CSS variable for the height that we just change the value of in this function, rather than having the `sticky_message_header_styles` element; seems fine to do that cleanup in a follow-up PR though.

Reply:
Oh, I see you in fact do this a couple commits later; I've amended that commit to delete `sticky_message_header_styles` from `index.html` when doing so, so we don't leak it.